lib/main.coffee

```diff
@@ -137,9 +152,14 @@ module.exports =
           return messages.map (message) ->
             startPoint = [message.line - 1, message.column - 1]
             endPoint = [message.line - 1, message.column]
-            return {
+            ret = {
               type: message.type
-              text: message.message
               filePath,
               range: [startPoint, endPoint]
             }
+            if showSource
+              ret.html = '<span class="badge badge-flexible">' + (message.source or 'Unknown') + '</span> '
```

Comment:
Do you know if there is a way I could link this to a description? I was actually going to make this an `<a>` tag with a link to the sniffs description but I can't find any documentation online.